Kinvey/Kinvey/DataStore.swift

```diff
@@ -9,28 +9,6 @@
 import Foundation
 import PromiseKit
 
-class DataStoreTypeTag: Hashable {
```

Comment:
Was the use of `DataStoreTypeTag` a cause of the memory leak?

Reply:
not really, again, i should create a separate PR for that. Doing that now.